packages/osd-stylelint-plugin-stylelint/src/rules/invalid_properties/index.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import stylelint from 'stylelint';
+import { NAMESPACE } from '../..';
+import {
+  getNotCompliantMessage,
+  getPropertyRule,
+  getRulesFromConfig,
+  isValidOptions,
+} from '../../utils';
+
+const { ruleMessages, report } = stylelint.utils;
+
+const ruleName = 'invalid_properties';
```

Comment:
nit: maybe `no_restricted_properties`? Because to me invalid property communicates that this property doesn't exist. But that's just my opinion.

Reply:
Yeah, I was having trouble coming up with a good name, thanks for the suggestion!